src/Symfony/Component/Routing/Matcher/UrlMatcher.php

```diff
@@ -91,6 +92,10 @@ public function match($pathinfo)
             return $ret;
         }
 
+        if ('/' === $pathinfo) {
```

Comment:
I would instead test if `0 === count($this->routes)`, which is more in line with what you've done in the matcher dumper and more accurate, right?

Reply:
Right :)

Reply:
But still we need both checks